storybook/stories/ui-charts/manchetteWithSpaceTimeChart/rectangle-zoom.stories.tsx

```diff
@@ -19,6 +19,7 @@ import { MouseTracker } from '../spaceTimeChart/helpers/components';
 
 import '@osrd-project/ui-core/dist/theme.css';
 import '@osrd-project/ui-charts/dist/theme.css';
+import '../../../styles/ManchetteWithSpaceTimeChart/rectangle-zoom.css'
```

Comment:
This breaks the package isolation (external users won't be able to perform this import), and I believe this style should be included in `@osrd-project/ui-charts/dist/theme.css`.

Reply:
Or is this style specific to stories? If so, it should be moved somewhere in `storybook/stories/ui-charts/`.

Reply:
this style is only used by the story, should it really belong to something we export to our users ?

Reply:
yes, it’s specific to the stories. I moved it